oml/metrics/embeddings.py

```diff
@@ -85,6 +86,7 @@ def __init__(
             cmc_top_k: Values of ``k`` to compute ``CMC@k`` metrics
             precision_top_k: Values of ``k`` to compute ``Precision@k`` metrics
             map_top_k: Values of ``k`` to compute ``MAP@k`` metrics
+            fmr_vals: Values of ``fmr`` to compute ``FNMR@FMR`` metrics
```

Comment:
let's mention that these values are quantiles here

Reply:
It could be confusing for users. What quantiles we are talking about? A user just need to evaluate FNMR@FMR, and quantiles are just a tool to archive this goal. So, we literally set desired values of FMR, and by coincidence this values are the quantiles of negative distances.

Reply:
Hm, I got your point of view. Should we at least mention it in the docs?
> we literally set desired values of FMR, and by coincidence this values are the quantiles of negative distances.

Reply:
What do you think about the following?
```
fmr_vals: Values of ``fmr`` (measured in per cents) to compute ``FNMR@FMR`` metrics for
```

Reply:
Hm, what do you think about a more detailed description?

Values of ``fmr`` (measured in percents) for which we compute the corresponding ``FNMR``. For example, if ``fmr_values = (20, 40)`` we will calculate ``FNMR@FMR=20`` and ``FNMR@FMR=40``.